Audacia.Log.AspNetCore/ServiceCollectionExtensions.cs

```diff
@@ -136,16 +149,17 @@ public static IServiceCollection ConfigureDependencyBodyContentLogging(
         services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
 
         // Configure global configuration for the LogActionFilter
-        services.Configure<LogActionFilterConfig>(configuration.GetSection("LogActionFilter"));
+        services.Configure<LogActionFilterConfig>(configuration.GetSection("LogDependencyFilter"));
```

Comment:
The generic should be `LogDependencyFilterConfig`

Reply:
Okay - updated.